### ISSUE TITLE
Force a specific mirror for MiKTeX package

### ISSUE THREAD

**Issue by MHDBNS** [MiKTeX/miktex#905]
### Discussed in https://github.com/MiKTeX/miktex/discussions/898

<div type='discussions-op-text'>

<sup>Originally posted by **MHDBNS** August 10, 2021</sup>
Hi,

Here is my problem. I have installed basic MiKTeX from an AWS EC2 Windows. For security reason we can only whitelist some websites. Is it possible to force a mirror and then whitelist it so I can download the packages I need.
Hope my issue is well described. If someone can help it could be great.

Thanks

Mhd</div>

**Comment by edocevoli** [comment]
You can select a specific mirror:

```sh
mpm --set-repository=URL
```

Hope this helps

**Comment by MHDBNS** [comment]
Hi,

Thanks for your quick reply. It works it proposed me to install a package following the repo I forced. However I can observe a new error and I'm not sure I understand it. Here is the log I got following the installation of the package.

Any idea that can help me solve this issue is very welcome :)

2021-08-19 11:20:09,815+0200 INFO  pdflatex - allowing known shell commands
2021-08-19 11:20:11,253+0200 WARN  pdflatex.packagemanager - dependancy problem: tools is required by _miktex-latex-required
2021-08-19 11:20:11,253+0200 WARN  pdflatex.packagemanager - dependancy problem: miktex-doc-2.9 is required by _miktex-packages-2.9
2021-08-19 11:20:14,738+0200 INFO  pdflatex - installing package tree-dvips triggered by tex/latex/tree-dvips\tree-dvips.sty
2021-08-19 11:20:14,754+0200 INFO  pdflatex.packagemanager - going to download https://api2.miktex.org/hello
2021-08-19 11:20:14,754+0200 FATAL pdflatex.core - SSL connect error
2021-08-19 11:20:14,754+0200 FATAL pdflatex.core - Data: code="35", url="https://api2.miktex.org/hello"
2021-08-19 11:20:14,754+0200 FATAL pdflatex.core - Source: Libraries\MiKTeX\PackageManager\CurlWebSession.cpp:247
2021-08-19 11:20:14,754+0200 FATAL pdflatex - SSL connect error
2021-08-19 11:20:14,754+0200 FATAL pdflatex - Info: code="35", url="https://api2.miktex.org/hello"
2021-08-19 11:20:14,754+0200 FATAL pdflatex - Source: Libraries\MiKTeX\PackageManager\CurlWebSession.cpp
